chalk-ir/src/lib.rs

```diff
@@ -942,6 +951,23 @@ pub struct ConstData<I: Interner> {
     pub value: ConstValue<I>,
 }
 
+impl<I: Interner> ConstData<I> {
+    /// Tries to evaluate the constant.
+    /// This is guaranteed to return a concrete constant or an error.
+    pub fn try_eval(&self, interner: &I) -> Result<ConstData<I>, ConstEvalError> {
```

Comment:
Why have this function? Instead of just `try_eval` on `Unevaluated`?

Reply:
Oops, I had this for convenience before and some changes I made removed all the calls I made to this function.